etl/steps/data/garden/urbanization/2024-01-26/ghsl_degree_of_urbanisation.meta.yml

```diff
@@ -0,0 +1,206 @@
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_population_suburbs_or_peri_urban_area:
+        title: Population in suburbs or peri-urban areas (DEGURBA Level 2)
+        unit: people
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_population_village:
+        title: Population of villages (DEGURBA Level 2)
+        unit: people
+        description_from_producer: Population of spatial units classified as Villages in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_city:
+        title: Number of cities (DEGURBA Level 2)
+        unit: cities
+        description_from_producer: Number of spatial units classified as Cities in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_dense_town:
+        title: Number of dense towns (DEGURBA Level 2)
+        unit: dense towns
+        description_from_producer: Number of spatial units classified as Dense Towns in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_mostly_uninhabited_area:
+        title: Number of mostly uninhabited areas (DEGURBA Level 2)
+        unit: uninhabited areas
+        description_from_producer: Number of spatial units classified as Mostly Uninhabited Areas in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_rural_dispersed_area:
+        title: Number of rural dispersed areas (DEGURBA Level 2)
+        unit: dispersed areas
+        description_from_producer: Number of spatial units classified as Rural Dispersed Areas in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_semi_dense_town:
+        title: Number of semi-dense towns (DEGURBA Level 2)
+        unit: towns
+        description_from_producer: Number of spatial units classified as Semi-Dense Towns in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_suburbs_or_peri_urban_area:
+        title: Number of suburbs or peri-urban areas (DEGURBA Level 2)
+        unit: suburbs or peri-urban areas
+        description_from_producer: Number of spatial units classified as Suburbs or Peri-Urban Areas in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+      degurba_l2_units_village:
+        title: Number of villages (DEGURBA Level 2)
+        unit: 'villages'
+        description_from_producer: Number of spatial units classified as Villages in Degree of Urbanisation level 2.
+        display:
+          numDecimalPlaces: 0
+
+
+      share_of_urban_population:
+        title: Share of total population in administrative units classified as urban (cities, towns and suburbs)
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 1
+
+      total_area_km2:
```

Comment:
Is this the same as total land area of each country?

Reply:
yes! thanks!